doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -1,84 +1,92 @@
-==========================
-BlueStore Config Reference
-==========================
+==================================
+ BlueStore Configuration Reference 
+==================================
 
 Devices
 =======
 
-BlueStore manages either one, two, or (in certain cases) three storage
-devices.
+BlueStore manages either one, two, or in certain cases three storage devices.
 
-In the simplest case, BlueStore consumes a single (primary) storage device.
-The storage device is normally used as a whole, occupying the full device that
-is managed directly by BlueStore. This *primary device* is normally identified
-by a ``block`` symlink in the data directory.
+In the simplest case, BlueStore consumes a single storage device. The device is
+known as the *primary device*. Any device used by BlueStore will be used
+entirely by BlueStore and nothing other than BlueStore. The primary device is
+identified by a ``block`` symlink in the data directory.
 
-The data directory is a ``tmpfs`` mount which gets populated (at boot time, or
-when ``ceph-volume`` activates it) with all the common OSD files that hold
-information about the OSD, like: its identifier, which cluster it belongs to,
-and its private keyring.
+The data directory is a ``tmpfs`` mount. When this data directory is booted or
```

Comment:
I want to call this the *metadata* directory, but if *data* is entrenched, so be it.